test/python/unit/api/test_sort.py

```diff
@@ -0,0 +1,714 @@
+#####################
+## test_sort.py
+## Top contributors (to current version):
+##   Yoni Zohar
+## This file is part of the CVC4 project.
+## Copyright (c) 2009-2021 by the authors listed in the file AUTHORS
+## in the top-level source directory and their institutional affiliations.
+## All rights reserved.  See the file COPYING in the top-level source
+## directory for licensing information.
+##
+## ########################################################
+##
+## Unit tests for sort API.
+## Obtained by translating test/unit/api/sort_black.cpp
+
+import pytest
+import pycvc4
+from pycvc4 import kinds
+from pycvc4 import Sort
+
+
+@pytest.fixture
+def solver():
+    return pycvc4.Solver()
+
+
+def create_datatype_sort(solver):
+    intSort = solver.getIntegerSort()
+    # create datatype sort to test
+    dtypeSpec = solver.mkDatatypeDecl("list")
+    cons = solver.mkDatatypeConstructorDecl("cons")
+    cons.addSelector("head", intSort)
+    cons.addSelectorSelf("tail")
+    dtypeSpec.addConstructor(cons)
+    nil = solver.mkDatatypeConstructorDecl("nil")
+    dtypeSpec.addConstructor(nil)
+    dtypeSort = solver.mkDatatypeSort(dtypeSpec)
+    return dtypeSort
+
+
+def create_param_datatype_sort(solver):
+    sort = solver.mkParamSort("T")
+    paramDtypeSpec = solver.mkDatatypeDecl("paramlist", sort)
+    paramCons = solver.mkDatatypeConstructorDecl("cons")
+    paramNil = solver.mkDatatypeConstructorDecl("nil")
+    paramCons.addSelector("head", sort)
+    paramDtypeSpec.addConstructor(paramCons)
+    paramDtypeSpec.addConstructor(paramNil)
+    paramDtypeSort = solver.mkDatatypeSort(paramDtypeSpec)
+    return paramDtypeSort
+
+
+def test_operators_comparison(solver):
+    try:
+        solver.getIntegerSort() == Sort(solver)
+        solver.getIntegerSort() != Sort(solver)
+        solver.getIntegerSort() < Sort(solver)
+        solver.getIntegerSort() <= Sort(solver)
+        solver.getIntegerSort() > Sort(solver)
+        solver.getIntegerSort() >= Sort(solver)
+    except RuntimeError:
+        pytest.fail()
+
+
+def test_is_boolean(solver):
+    assert (solver.getBooleanSort().isBoolean())
```

Comment:
```suggestion
    assert solver.getBooleanSort().isBoolean()
```

Don't need the parentheses for an assert.

Reply:
Fixed in 5daa7a5 for all asserts.